public/app/features/explore/ExploreQueryInspector.tsx

```diff
@@ -59,9 +65,14 @@ export function ExploreQueryInspector(props: Props) {
         data={dataFrames}
         dataName={'Explore'}
         isLoading={queryResponse.state === LoadingState.Loading}
-        options={{ withTransforms: false, withFieldConfig: false }}
+        options={dataOptions}
         timeZone={timeZone}
         app={CoreApp.Explore}
+        formattedDataDescription="Matches the format in the panel"
+        onOptionsChange={setDataOptions}
+        // need these props so formatting can work
+        panelPluginId={exploreId}
```

Comment:
Decided to just `exploreId` to the `panelPluginId` prop although the intention is to pass id that is basically the type of panel, and the main purpose of `panelPluginId` is in this function that checks if `panelPluginId='table'` and if it is then removes any custom table display settings: https://github.com/grafana/grafana/blob/dd77ff6bcde9f363927795ef70769989d7b22181/public/app/features/inspector/InspectDataTab.tsx#L197

In Explore we don't care about this because we don't support any custom table display settings.

Reply:
Good point. Explore shows inspector for all data frames (not for a specific panel). Maybe we can change the logic that checks for panel id and do the clean up only if it's provided?

e.g.

```typescript
  getProcessedData(): DataFrame[] {
    const { options, panelPluginId, fieldConfig, timeZone } = this.props;
    const data = this.state.transformedData;

    if (!options.withFieldConfig) {
      return applyRawFieldOverrides(data);
    }

    const fieldConfigCleaned = panelPluginId && fieldConfig ? this.cleanTableConfigFromFieldConfig(panelPluginId, fieldConfig) : fieldConfig;
...
```

@grafana/dashboards-squad wdyt?